__tests__/getWeatherFormatted.test.ts

```diff
@@ -0,0 +1,19 @@
+import {getWeatherData} from '../API/getWeatherData';
+
+describe('get weather', () => {
+  it('Api test', async () => {
+    global.fetch = jest.fn().mockImplementation(() => {
+      var p = new Promise(resolve => {
+        resolve({
+          json: function () {
+            return {city: 'Orgerus'};
+          },
+        });
+      });
+      return p;
+    });
+
+    const response = await getWeatherData();
+    expect(response.city).toBe('Orgerus');
```

Comment:
Here, we more expect to see the "whole flow" like :
Your mock returns not just a city but the whole rawDataForTest.ts 
you can also expect function calls

So : 
1) mock fetch returning server-like data, not just city
2) expect fectch to be called
3) expect the output to be formatted data